api/trace/src/main/java/io/opentelemetry/spi/trace/TracerProviderFactory.java

```diff
@@ -5,7 +5,6 @@
 
 package io.opentelemetry.spi.trace;
```

Comment:
I moved this file into the `api-trace` folder for now. Later we'd presumably move all the SPI into an `autoconfigure` or such artifact based on #2262